operator-framework/src/test/java/io/javaoperatorsdk/operator/sample/standalonedependent/StandaloneDependentTestReconciler.java

```diff
@@ -44,8 +42,8 @@ public StandaloneDependentTestReconciler() {
 
   @Override
   public void setKubernetesClient(KubernetesClient kubernetesClient) {
+    configMapDependent = new DeploymentDependentResource(kubernetesClient);
```

Comment:
This looks/feels weird…

Reply:
well, the way around is put back the setter :)

Reply:
To me, that's a smell in the design…

Reply:
Where exactly? in dependent resource side, or in the test?